utilities/rpp-performancetests/HOST_NEW/Tensor_host_pkd3.cpp

```diff
@@ -1156,6 +1159,57 @@ int main(int argc, char **argv)
 
             break;
         }
+        case 30:
```

Comment:
looks like case 30 is repeated here

Reply:
This one is actually needed, we need the name of the functionality being run first - To decide all the u8/f32/f16/i8 or if the case being run is special (like has interpolationType or filterKernelSize). After all params are locked to the variant being run (with common code), there is another switch case to actually run the RPP call.